.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleAttributePerLine": true,
+  "trailingComma": "es5"
```

Comment:
Se per ora ci aspettiamo di avere solo file markdown, ci servono queste due regole? Male non fanno, ma probabilmente non hanno nemmeno effetto ad oggi.

Il file (anche vuoto) se non sbaglio serve comunque sennò potrebbero venir presi dei default dal config locale (o almeno, ho memoria di questa cosa ma non trovo evidenza in giro, quindi magari mi ricordo male)

Reply:
Ho lo stesso ricordo a riguardo sinceramente quindi ci può stare toglierle

Reply:
Il file, anche vuoto, è necessario in quanto altrimenti Prettier lo cerca in alto nel tree fino a trovarne uno valido: ciò potrebbe portare ad inconsistenze.
Mi sono permesso di aggiungere queste 2 regole che, è vero non hanno effetto sul markdown, ci troveremo avanti SE ci dovesse essere del codice da formattare "dentro" il markdown.

Se le riteniamo useless, si possono togliere tranquillamente comunque.

cc @Cadienvan

Reply:
Figurati, a esposizione!

Reply:
Ok questa spiegazione convince anche me e ha molto senso visto che sicuro anche solo uno snippet per sbaglio ci sarà! Come dicevo aspetto ancora qualche giorno e poi la portiamo dentro!

Reply:
> Bene quindi almeno il file vuoto siamo d'accordo che serve :)

Giusto per i poster, quando parlo di file vuoto in questo caso intendo un file con un `empty object {}` almeno.